Doc/library/time.rst

```diff
@@ -225,10 +228,15 @@ Functions
 
 .. function:: ctime([secs])
 
-   Convert a time expressed in seconds since the epoch to a string representing
-   local time. If *secs* is not provided or :const:`None`, the current time as
-   returned by :func:`.time` is used.  ``ctime(secs)`` is equivalent to
-   ``asctime(localtime(secs))``. Locale information is not used by :func:`ctime`.
+   Convert a time expressed in seconds since the epoch to a string of a form:
+   ``'Sun Jun 20 23:21:05 1993'`` representing local time. The day field
+   is two characters long and is space padded if the day is a single digit,
+   e.g.: ``'Wed Jun  9 04:26:40 1993'``.
+
+   If *secs* is not provided or
+   :const:`None`, the current time as returned by :func:`.time` is used.
```

Comment:
There's an unnecessary line break here. Can you please rewrap the paragraph  (I mean by sticking under 80ish columns and without random newlines in it)?

Anything between:
```
If *secs* is not provided or :const:`None`, the current time as
returned by :func:`.time` is used. ``ctime(secs)`` is equivalent to
``asctime(localtime(secs))``.  Locale information is not used by
:func:`ctime`.
```

and

```
If *secs* is not provided or :const:`None`, the current time as returned by
:func:`.time` is used. ``ctime(secs)`` is equivalent to ``asctime(localtime(secs))``.
Locale information is not used by :func:`ctime`.
```
would work for me. I personally tend to be conservative (like my first example), so future small edits won't always break 80ish columns, allowing to *not* rewrap the paragraph, yielding easy-to-read single-line diffs.  In my second example, almost any edit would force one to rewrap the paragraph, yielding to a harder to read big diff.